include/deal.II/base/quadrature_point_data.h

```diff
@@ -328,7 +328,7 @@ namespace parallel
     {
     public:
       DEAL_II_STATIC_ASSERT(std::is_base_of<TransferableQuadraturePointData, DataType>::value,
-                            "User's DataType class should be derived from QPData");
+                            "User's DataType class should be derived from QPData")
```

Comment:
qtcreator shows a warning about "extra ';'' and underlines it without my changes (probably because it doesn't see ``DEAL_II_WITH_CXX11``). It handles this new code just fine.